docker/Dockerfile

```diff
@@ -11,6 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+FROM centos:centos7 AS dist
+ARG PRESTO_VERSION
+ARG CLIENT_VERSION=${PRESTO_VERSION}
```

Comment:
i know its preexisting, but why do we have two ARGs instead of one?

Reply:
I don't now, I guess @11xor6 thought you might want to specify a different client and server version.

Reply:
hmm... why would we want to bundle different versions on a docker image? @11xor6?